Code/GraphMol/MolDraw2D/MolDraw2D.cpp

```diff
@@ -1068,6 +1069,10 @@ pair<string, MolDraw2D::OrientType> MolDraw2D::getAtomSymbolAndOrientation(
     symbol = "";
   } else if (isComplexQuery(&atom)) {
     symbol = "?";
+  } else if (drawOptions().atomLabelDeuteriumTritium && atom.getAtomicNum() == 1 &&
```

Comment:
It's definitely worth thinking about adding something like that for the C++ code, though I'd probably do it with a function instead of adding a method to the options class.